ObjCVideoQuickstart/ViewController.m

```diff
@@ -142,24 +138,28 @@ - (void)startPreview {
 }
 
 - (void)flipCamera {
-    if (self.camera.source == TVIVideoCaptureSourceFrontCamera) {
-        [self.camera selectSource:TVIVideoCaptureSourceBackCameraWide];
+    if (self.camera.source == TVICameraCaptureSourceFrontCamera) {
+        [self.camera selectSource:TVICameraCaptureSourceBackCameraWide];
     } else {
-        [self.camera selectSource:TVIVideoCaptureSourceFrontCamera];
+        [self.camera selectSource:TVICameraCaptureSourceFrontCamera];
     }
 }
 
 - (void)prepareLocalMedia {
```

Comment:
I'm not sure that it needs to be changed. The tracks are collectively the local media which we are sharing.